DiabloII.Application.Tests/TestDependencies.cs

```diff
@@ -24,10 +24,12 @@ public static IServiceCollection CreateServices()
             var services = testContext.Services;
 
             services
+                .AddSingleton(services)
                 .AddSingleton(testContext.DbContext)
                 .AddSingleton<IHttpService>(testContext.HttpService)
                 .AddSingleton<IUsersTestContext, UsersTestContext>()
                 .AddSingleton<ISuggestionsTestContext, SuggestionsTestContext>()
+                .AddSingleton<IHalSuggestionsTestContext, HalSuggestionsTestContext>()
                 .AddSingleton<INotificationsTestContext, NotificationsTestContext>()
                 .AddSingleton<ISuggestionsRepository, SuggestionsRepository>()
                 .AddSingleton<IUserRepository, UserRepository>()
```

Comment:
Most of those things should be deduce by the an IOC lib by scanning my assemblies.

Reply:
done 👍